### DISCUSSION TITLE
What device is currently supported best for a new project, how is expressobin?

### DISCUSSION THREAD

**Discussion by ioogithub** [Ysurac/openmptcprouter#2496] [General]
The expressobin caught my attention, it seems like an ideal router as it has two physical LAN ports and 1 WAN port. I can't find any discussion or details of the current state of support for this device. Is there a changelog or list of features that are working/not working?

The github page doesn't have an experimental warning ike many of the other boards (https://github.com/Ysurac/openmptcprouter/wiki/Router-install) however on the website it does say beta (https://www.openmptcprouter.com/download).

How beta is the expressobin, is everything working. Would this be a good candidate for a new openmptrouter project? If the software support is good enough it would have an advantage of the raspberry pi in that a separate switch would not be needed.

**Comment by Ysurac** [comment]
Some user have it working correctly, for others it's really too slow.
In every case, espressobin CPU is slow, so max speed is really low.
Best device is always x86_64 based, but others devices can work great too if you know max aggregated speed. A RPI4 can deliver an aggregated speed of 350Mb/s, so every CPU slower will do less than that.

**Reply by ioogithub** [reply]
Thanks this is very helpful. I would prefer to go with a lower powered arm if I can get a first class experience. Of the working arm based solutions, does the raspberry pi have the strongest processor?

I am looking at aggregating two lines with a total bandwidth of 170MB/s maximum but more realistically around 100MB/s. In this case would you still recommend against the expressobin? I see the bananna pi also is also beta, is this because of processor constraints as well?

I like the idea of a raspberry pi and glad to hear it can potentially handle my application. I saw another user mention expanding the Pi's lan ports with a TP-LINK UE330. Would it be a viable solution to use this usb hub to connect the two aggregate lines or would the pi's shared usb bandwidth be a bottleneck?
